shared/wallets/common/markdown-memo.tsx

```diff
@@ -40,7 +40,7 @@ const MarkdownMemo = (props: Props) =>
       {!props.hideDivider && <Kb.Divider vertical={true} style={styles.quoteMarker} />}
       <Kb.Markdown
         style={styles.memo}
-        styleOverride={Styles.collapseStyles([isMobile ? styleOverride : undefined, props.styleOverride])}
```

Comment:
oh i didn't realize you need to mix on mobile. i'd do
```
styleOverride={{
...(isMobile ? styleOverride : {}),
...props.styleOverride
}}
```

Reply:
done